wiki/Help_centre/Store/Store_archive/de.md

```diff
@@ -12,7 +12,7 @@ tags:
   - Tastatur
 ---
 
-# Store archive
+# Shoparchiv
 
 *Hauptseite: [osu!store und Merchandise](/wiki/Help_centre/Store)*\
 *Anmerkung: Diese Produkte werden nicht mehr länger zum Verkauf im osu!store angeboten.*
```

Comment:
```suggestion
*Anmerkung: Diese Produkte werden nicht mehr zum Verkauf im osu!store angeboten.*
```
idk if we should keep osu!store as a term or change it to something like osu!store, osu!laden...

Reply:
According to the [Brand identity guidelines](https://osu.ppy.sh/wiki/de/Brand_identity_guidelines), `osu!store` is a protected name and must not include spaces.

I would like to stick to the original name where possible due to this.

Reply:
> According to the [Brand identity guidelines](https://osu.ppy.sh/wiki/de/Brand_identity_guidelines), `osu!store` is a protected name and must not include spaces.
> 
> I would like to stick to the original name where possible due to this.

Sounds good to me. The context should be enough for anybody to understand what it is although we could turn osu!store into a link to https://osu.ppy.sh/store/listing

Reply:
```suggestion
*Anmerkung: Diese Produkte werden nicht mehr zum Verkauf im osu!store angeboten.*
```